docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -47,21 +51,27 @@ A DaemonSet also needs a [`.spec`](https://git.k8s.io/community/contributors/dev
 
 ### Pod Template
 
-The `.spec.template` is the only required field of the `.spec`.
+The `.spec.template` is one of the required fields `.spec`.
```

Comment:
`.spec.template` and `.spec.selector` are required fields? I realize as per the types.go, only the former is "required", but we block with validation an empty field in the latter as well.

Reply:
In the pod selector section I added details about defaulting and its removal in 1.8